cmd/certsuite/info/info.go

```diff
@@ -0,0 +1,89 @@
+package info
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/identifiers"
+	"github.com/test-network-function/cnf-certification-test/internal/cli"
+	"github.com/test-network-function/test-network-function-claim/pkg/claim"
+)
+
+var (
+	infoCmd = &cobra.Command{
+		Use:   "info",
+		Short: "Displays information from the test catalog",
+		RunE:  showInfo,
+	}
+)
+
+func showInfo(cmd *cobra.Command, _ []string) error {
+	testCaseFlag, _ := cmd.Flags().GetString("test-case")
+
+	var testCase claim.TestCaseDescription
+	for id := range identifiers.Catalog {
+		if id.Id == testCaseFlag {
+			testCase = identifiers.Catalog[id]
+			break
+		}
+	}
+
+	if testCase.Identifier.Id == "" {
+		return fmt.Errorf("no test case found matching name %q", testCaseFlag)
+	}
+
+	const lineMaxWidth = 120
```

Comment:
Optional: get the width from terminal to properly wrap lines when the terminal is too small
```
package main

import "golang.org/x/term"

func main() {
    if term.IsTerminal(0) {
        println("in a term")
    } else {
        println("not in a term")
    }
    width, height, err := term.GetSize(0)
    if err != nil {
        return
    }
    println("width:", width, "height:", height)
}
```

Reply:
Done.